sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentResult.cs

```diff
@@ -38,5 +38,11 @@ public DocumentSentiment DocumentSentiment
                 return _documentSentiment;
             }
         }
+
+        /// <inheritdoc/>
+        public override string ToString()
+        {
+            return $"ID: {Id}, {DocumentSentiment}";
```

Comment:
As an example showing most important properties that help differentiate instances, and elements in collections:
![image](https://user-images.githubusercontent.com/9868623/95146851-d4cf1600-0733-11eb-9acf-73e2e887aff0.png)